lib/samples/find_closest_facility_from_point/find_closest_facility_from_point_sample.dart

```diff
@@ -66,14 +75,15 @@ class _FindClosestFacilityFromPointSampleState
                 crossAxisAlignment: CrossAxisAlignment.center,
                 mainAxisAlignment: MainAxisAlignment.spaceEvenly,
                 children: [
+                  // Create buttons to solve the routes and reset the graphics.
                   ElevatedButton(
-                    onPressed: !_isRouteSolved && _isInitialized
+                    onPressed: !_routeSolved && _initialized
```

Comment:
With other samples, we have gone with a view-spanning loading overlay until `_ready` is set instead of deactivating individual controls.

Reply:
Okay!